tests/integration/EntryPoints/REST/UnapprovePageApiTest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace ProfessionalWiki\PageApprovals\Tests\EntryPoints\REST;
+
+use MediaWiki\Rest\RequestData;
+use MediaWiki\Tests\Rest\Handler\HandlerTestTrait;
+use MediaWiki\Tests\Unit\Permissions\MockAuthorityTrait;
+use ProfessionalWiki\PageApprovals\EntryPoints\REST\UnapprovePageApi;
+use ProfessionalWiki\PageApprovals\Tests\PageApprovalsIntegrationTest;
+
+/**
+ * @covers \ProfessionalWiki\PageApprovals\EntryPoints\REST\UnapprovePageApi
+ * @group database
+ */
+class UnapprovePageApiTest extends PageApprovalsIntegrationTest {
+	use HandlerTestTrait;
+	use MockAuthorityTrait;
```

Comment:
What is `MockAuthorityTrait` for?

Reply:
It's going to be needed for tests to set the authorities/uses. It was added a bit prematurely here.